authority/admin/db.go

```diff
@@ -71,6 +71,52 @@ type DB interface {
 	DeleteAdmin(ctx context.Context, id string) error
 }
 
+type NoDB struct{}
```

Comment:
You're using this only on the cli, why don't just have this there as a private or internal type.

Reply:
I think we want to remove this. Or as I suggested on the cli, we might prefer an admin.DB implementation that works directly on the ca.json